Ansible-security.md

```diff
@@ -0,0 +1,10 @@
+
+1. Ansible Server:
```

Comment:
I'm unaware of "Ansible Server". Do you mean [Ansible control node](https://docs.ansible.com/ansible/latest/network/getting_started/basic_concepts.html#control-node)?

Reply:
Yes cristian , I meant ansible control node